release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -625,6 +625,13 @@ For more information, see xref:../networking/network_observability/network-obser
 [id="ocp-4-12-storage"]
 === Storage
 
+[id="ocp-4-12-low-latency-tuning"]
+=== Low latency tuning
+
+[id="ocp-4-12-low-latency-tuning-features-options"]
+==== Support for new kernel features and options
+The low latency tuning has been updated to use the latest kernel features and options. The fix for link:https://bugzilla.redhat.com/show_bug.cgi?id=2117780[2117780] introduced a new per-CPU `kthread`, `ktimers`. This thread must be pinned to the proper CPU cores. With this update, there is no functional; the isolation of the workload is the same. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2102450 [2102450].
```

Comment:
```suggestion
The low latency tuning has been updated to use the latest kernel features and options. The fix for link:https://bugzilla.redhat.com/show_bug.cgi?id=2117780[2117780] introduced a new per-CPU `kthread`, and `ktimers`. This thread must be pinned to the proper CPU cores. With this update, there is no functional. The isolation of the workload is the same. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2102450 [2102450].
```
In the sentence that starts with 'With this update...', there seems to be something missing after 'functional'.